Documentation/gettingstarted/minikube.rst

```diff
@@ -67,13 +67,9 @@ Kubernetes CNI plugin.
 
 .. parsed-literal::
 
-    kubectl create -f \ |SCM_WEB|\/install/kubernetes/quick-install.yaml
+    kubectl create -f \ |SCM_WEB|\/install/kubernetes/experimental-install.yaml
```

Comment:
I think we should still default to `quick-install.yaml` and provide `experimental-install.yaml` as an option in a box below, explaining what it does. I think it looks a bit scary for a "Getting Started" guide to immediately recommend installing features which are explicitly marked not ready for production use.